### ISSUE TITLE
Toggle switch for entity holds focus until clicked away

### ISSUE THREAD

**Issue by mwasowski** [home-assistant/frontend#5898]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

I noticed recently that when I click an entity in card group (or single entity for that matter), the element keeps focus until anything else is clicked making it loose focus as seen on the example:

## Expected behavior

When the toggle is clicked the entity looses focus.

## Steps to reproduce

1. Add entity card with any type of a switch/input boolean
2. Change it's state.

## Environment

- Home Assistant release with the issue: HA 0.109.6, HassOS 3.13, Supervisor 222
- Last working Home Assistant release (if known): -
- Browser and browser version: Firefox 76.0.1 x64
- Operating system: Windows 10 x64 Build 1909

## State of relevant entities

It's not relevant to specific entity but I noticed it after I added this.
```yaml
entities:
  - entity: sensor.bridge_state
  - entity: input_boolean.zigbee_permit_join
  - entity: timer.zigbee_permit_join
show_header_toggle: false
title: Zigbee
type: entities
```
![JfI0OFu6X2](https://user-images.githubusercontent.com/1921808/82075690-eceabe00-96dc-11ea-8c7f-31866fde9427.gif)

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

**Comment by bramkragten** [comment]
This is just how websites work, if you click something it gets focus.

**Comment by mwasowski** [comment]
I have to disagree or maybe I'll rephrase. The 'loosing focus' is referring to the visual aspect. The animation looks like it's stuck half way through with the round shadow around the control.

**Comment by mwasowski** [comment]
Why this was closed without even taking into consideration or allowing me to explain. This is a bug, maybe visual one but it is.

**Comment by bramkragten** [comment]
We just use and follow material elements/guidelines. We are not going to make changes to the behaviour of those elements.

You can create an issue upstream if you disagree.

**Comment by mwasowski** [comment]
Ok, this indeed might be a correct implementation according to Material Design Guidelines. This is just beyond me, I did not think that it might be the case:
![FtNunW3Vjl](https://user-images.githubusercontent.com/1921808/82084576-c6804f00-96eb-11ea-8e88-454419c6fa3f.gif)
That's from:
https://material-components.github.io/material-components-web-catalog/#/component/switch
it behaves the same way HA toggle does.

So  you were right, I think I just never realized this and never encountered this, maybe because I have more experience with Material Design in Angular and there it behaves as I was expecting, the ripple effect disappears after the action finishes:
![AKjHpB01jH](https://user-images.githubusercontent.com/1921808/82084545-ba948d00-96eb-11ea-822a-6b454fad48a7.gif)
That's from:
https://material.angular.io/components/slide-toggle/overview

Sorry for opening another issue, but I was genuinely triggered. I cooled down a bit ;)
